examples/riot-kitchen-sink/riotshots.test.js

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
+
+initStoryshots({
```

Comment:
I don't see any implementation for `riot` in `storyshots`. Is it something you are going to work on?

Reply:
You can take as an example html or svelte implementation for storyshots.

Reply:
Does not work.

Reply:
What is the command line to proceed to the record ?

Reply:
done I think